<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Đăng Nhập</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>721</width>
      <height>621</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/background-thoi-trang-cho-be_025700025.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>421</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{
	background-color:rgb(255, 158, 158);
	border-radius:25px
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>5</y>
       <width>421</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Đăng Nhập&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLogin">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>250</y>
       <width>321</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font-size: 20px;
    border-radius: 10px;
	background-color: rgb(0, 85, 255);
	color:rgb(255, 255, 255);
	font-weight: bold;
}</string>
     </property>
     <property name="text">
      <string>Đăng Nhập</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtName">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>321</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
QLineEdit{
	border-radius: 5px;
}</string>
     </property>
     <property name="placeholderText">
      <string>Tên tài khoản...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPass">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>160</y>
       <width>321</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Nhập mật khẩu...</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
